packages/@okta/okta-auth-js/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@okta/okta-auth-js",
+  "description": "The Okta Auth SDK",
+  "version": "2.10.0",
+  "homepage": "https://github.com/okta/okta-auth-js",
+  "license": "Apache-2.0",
+  "main": "lib/server/serverIndex.js",
+  "browser": "lib/browser/browserIndex.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/okta/okta-auth-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/okta/okta-auth-js/issues"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml .",
+    "test": "yarn test:karma && yarn test:browser && yarn test:server",
+    "test:karma": "karma start --single-run",
+    "test:browser": "jest --config ./jest.browser.js",
+    "test:server": "jest --config ./jest.server.js",
+    "test:report": "yarn test --ci --silent || true",
+    "build": "node ./writeConfig.js && webpack --config webpack.config.js",
+    "prepare": "yarn build"
+  },
+  "author": "Okta",
+  "keywords": [
+    "Okta",
+    "authentication",
+    "auth",
+    "login"
+  ],
+  "dependencies": {
+    "Base64": "0.3.0",
+    "cross-fetch": "^3.0.0",
+    "js-cookie": "2.2.0",
+    "node-cache": "^4.2.0",
+    "q": "1.4.1",
+    "reqwest": "2.0.5",
+    "tiny-emitter": "1.1.0",
+    "xhr2": "0.1.3"
+  },
+  "devDependencies": {
+    "babel-jest": "^24.9.0",
+    "eslint": "5.6.1",
+    "istanbul-instrumenter-loader": "^3.0.1",
+    "jasmine-ajax": "^4.0.0",
+    "jest": "^24.9.0",
+    "jest-junit": "^9.0.0",
+    "jquery": "3.3.1",
+    "json-loader": "0.5.4",
+    "karma": "^4.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage-istanbul-reporter": "^2.0.5",
+    "karma-jasmine": "^2.0.1",
+    "karma-jquery": "^0.2.3",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^3.0.5",
+    "lodash": "4.17.11",
+    "promise.allsettled": "^1.0.1",
+    "promise.prototype.finally": "^3.1.1",
+    "webpack": "^3.0.0"
+  },
+  "okta-auth-js": {
+    "STATE_TOKEN_KEY_NAME": "oktaStateToken",
+    "DEFAULT_POLLING_DELAY": 500,
+    "DEFAULT_MAX_CLOCK_SKEW": 300,
+    "DEFAULT_CACHE_DURATION": 86400,
+    "FRAME_ID": "okta-oauth-helper-frame",
+    "REDIRECT_OAUTH_PARAMS_COOKIE_NAME": "okta-oauth-redirect-params",
+    "REDIRECT_STATE_COOKIE_NAME": "okta-oauth-state",
+    "REDIRECT_NONCE_COOKIE_NAME": "okta-oauth-nonce",
+    "TOKEN_STORAGE_NAME": "okta-token-storage",
```

Comment:
IIRC, this isn't used, so orthogonal to your discussion on whether to move these, I'd recommend ripping out the ones that don't exist anymore.

Reply:
👍 This refactor includes the generated `lib/config.js` checked in. I have added an issue for this work: https://oktainc.atlassian.net/browse/OKTA-259362